spec/requests/shopping_lists_spec.rb

```diff
@@ -47,22 +49,71 @@
         end
 
         context 'when only the new shopping list has been created' do
-          let!(:aggregate_list) { create(:aggregate_shopping_list, user: user, created_at: 2.seconds.ago, updated_at: 2.seconds.ago) }
+          let!(:aggregate_list) { create(:aggregate_shopping_list, game: game, created_at: 2.seconds.ago, updated_at: 2.seconds.ago) }
 
           it 'creates one list' do
-            expect{ create_shopping_list }.to change(user.shopping_lists, :count).from(1).to(2)
+            expect{ create_shopping_list }.to change(game.shopping_lists, :count).from(1).to(2)
           end
 
           it 'returns only the newly created list' do
             create_shopping_list
-            expect(response.body).to eq(user.shopping_lists.last.to_json)
+            expect(response.body).to eq(game.shopping_lists.last.to_json)
+          end
+        end
+
+        context 'when the request does not include a body' do
```

Comment:
The docs mention this behaviour so I thought it would be good to have tests for it as well.